dspace-api/src/main/java/org/dspace/storage/bitstore/README.wiki

```diff
@@ -0,0 +1,38 @@
+==== Cleanup ====
```

Comment:
Suggestion for next time:  attach a file like this to the Jira issue rather than placing it in the source tree.  We'll need to remove this file after the wiki is updated.